### ISSUE TITLE
Change Import tab to Import Settings when no Resources selected

### ISSUE THREAD

**Issue by mrjustaguy** [godotengine/godot-proposals#4314]
### Describe the project you are working on

General usage

### Describe the problem or limitation you are having in your project

Import Tab is useless when No Resources are Selected, as it just tells you to select a resource to adjust import settings.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

when No Resources are selected in the Import tab, allow for Modifying Importer Defaults for the supported File formats and the Import subcategories
Example: 
Supported File Format: .png
Import subcategories: Texture3D, Texture2D, Image.....

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

UI - Make it exactly like when you have a selected resource, just where it says resource name, you add an option button for the specific Format, and adjust everything according to the format selected.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

You can find and select a resource of the format which you want to change default importer options for.

### Is there a reason why this should be core and not an add-on in the asset library?

Importer is core.

**Comment by Calinou** [comment]
I think this can be confusing, especially as changing import settings is something that should _never_ happen on the wrong resources. I often tell beginners to change import settings, and it's very important that this process should _never_ go wrong.

**Comment by mrjustaguy** [comment]
Yeah didn't entirely think of that aspect... however beginners probably won't be tweaking this to begin with...
Maybe a good compromise would be to add a button below the current message that is "Change Default Import Settings" and aside from that works as above.
This way you won't accidentally change defaults because you have to press the button to initiate that step, but will be able to do so if you need to, without having to bang your head on the wall trying to figure out how to... Adding a Revert button for extra security probably wouldn't hurt..
